scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -2132,6 +2098,54 @@
                 "schedule each pool less often than the desired setting of every " target-per-pool-match-interval-millis " ms."))
     (async/pipe (chime-ch (util/time-seq (time/now) (time/millis match-interval-millis))) match-trigger-chan)))
 
+(defn make-pool-handler
+  "Make the configured handler for the pool to do scheduling."
+  [conn fenzo-state pool-name->pending-jobs-atom agent-attributes-cache fenzo-max-jobs-considered scaleback
+   floor-iterations-before-warn floor-iterations-before-reset trigger-chan rebalancer-reservation-atom
+   mesos-run-as-user pool-name cluster-name->compute-cluster-atom job->acceptable-compute-clusters-fn]
+  (let [fenzo (:fenzo fenzo-state)
+        resources-atom (atom (view-incubating-offers fenzo))
+        using-pools? (not (nil? (config/default-pool)))]
+    (reset! fenzo-num-considerable-atom fenzo-max-jobs-considered)
+    (tools/chime-at-ch
+     trigger-chan
+     (fn pool-schedule-event []
+       (log-structured/info "Starting pool handler" {:pool pool-name})
+       (prom/with-duration
+         prom/scheduler-match-cycle-duration {:pool pool-name}
+         (tracing/with-span [s {:name "scheduler.pool-handler" :tags {:pool pool-name :component tracing-component-tag}}]
```

Comment:
Spoke with Sam, they asked me to add a span here in addition to the span inside the Fenzo handler.